classes/Domain/Services/ResetEmailer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace OpenCFP\Domain\Services;
+
+class ResetEmailer
```

Comment:
You're on the right track. For this project, you might just be building a domain history of arbitrary use-cases and making it easily addressable. Sourcing entities from that history isn't necessarily required. You get the best of both worlds for as long as you want.